ops/docker/ci-builder/Dockerfile

```diff
@@ -11,14 +11,14 @@ RUN apt-get update && \
     chmod +x ./rustup.sh && \
     ./rustup.sh -y
 
-# move the foundryrc file to the foundry dir
-WORKDIR /opt/foundry
-COPY ../../.foundryrc ./.foundryrc
+COPY ./.foundryrc ./.foundryrc
```

Comment:
L18 fails otherwise as you can't clone into a non-empty directory. So we copy .foundryrc to /opt instead.

Reply:
Wierd how this ever worked

Reply:
it didn't 👅